admin_manual/conf.py

```diff
@@ -48,9 +48,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '10.0'
+version = '10.3'
```

Comment:
oh wait, shouldn't this stay "10.0" ?

Reply:
I thought we were changing it to 10.3?

Reply:
Sorry, I see my mistake. It's `10.0.3`, not `10.3`. I'll fix the original and create a new backport.

Reply:
and here it is just the "short X.Y version" so does not need changing at all